jso/apis/src/main/java/org/teavm/jso/dom/events/WheelEvent.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright 2014 Alexey Andreev.
```

Comment:
I'm not sure about legal issues with author's name, but, please, at least put recent year here